test/cypress/integration/widgets/deployment_button_spec.ts

```diff
@@ -512,13 +549,68 @@ describe('Create Deployment Button widget', () => {
                 cy.get('@text').should('have.text', 'en');
             });
 
-            cy.contains('.field', 'string_default').within(() => {
+            cy.getField('string_default').within(() => {
                 verifyTextInput('Some default string');
             });
 
-            cy.contains('.field', 'string_default_null').within(() => {
+            cy.getField('string_default_null').within(() => {
                 verifyTextInput('null');
             });
         });
+
+        it('textarea', () => {
+            selectBlueprintInModal('textarea');
+
+            cy.getField('textarea_default_en').within(() => {
+                verifyTextareaContent('en');
+            });
```

Comment:
Optional:

I see right now that we can save some space by doing the following:
```suggestion
            cy.getField('textarea_default_en').within(() => verifyTextareaContent('en'));
```
but I think that's personal preference, so I'm OK to keep it as is.

Reply:
I tested it locally on freshly configured cloudify-stage. It's passed.
![image](https://user-images.githubusercontent.com/97745174/160116835-0d3cb44c-d5ba-4206-a285-5a9f8183ed1f.png)

However it's not passing on the pipeline. Checked twice.

It can happen because of four reasons:
1. Different cloudify-manager image used. I don't use locally recent dev image version.
2. The other tests affects starting state of the test.
3. The test deploument_button_spec is bigger some and it cause some problems in our system tests.
About other failing tests:
4.  The modification I make in the test may affect the starting state for other tests.
5. I added some blueprints to  .zip file. It may affects other tests. I didn't investigate that too.

Despite of that I decided to merge it according to your advice sent on slack. https://cloudify.slack.com/archives/G9MN0TDG8/p1648206850765449?thread_ts=1648206302.899609&cid=G9MN0TDG8